src/main/java/bio/terra/service/resourcemanagement/ResourceService.java

```diff
@@ -103,6 +118,86 @@ public GoogleBucketResource getOrCreateBucketForFile(Dataset dataset,
             flightId);
     }
 
+    /**
+     * Given an application deployment, get or create a storage account.
+     *
+     * @param dataset        dataset to create storage account for
+     * @param billingProfile authorized profile for billing account information case we need to
+     *                       create an application deployment or storage account
+     * @param flightId       used to lock the bucket metadata during possible creation
+     * @return a reference to the bucket as a POJO AzureStorageAccountResource
+     * @throws CorruptMetadataException in two cases.
+     * <ol>
+     *     <le>if the storage account already exists, but the metadata does not AND
+     *     the application property allowReuseExistingBuckets=false.</le>
+     *     <le>if the metadata exists, but the storage account does not</le>
```

Comment:
Not sure what `le` is but I don't think it will work. I'd use `li`.
```suggestion
     *     <li>if the storage account already exists, but the metadata does not AND
     *     the application property allowReuseExistingBuckets=false.</li>
     *     <li>if the metadata exists, but the storage account does not</li>
```
Also I'd use `ul` not `ol` but that's up to you.

Reply:
ah, copy pasta.  fixed elsewhere too